source/_integrations/ebusd.markdown

```diff
@@ -50,7 +52,7 @@ name:
 monitored_conditions:
   description: List of conditions to monitor. Note that not all monitored_conditions listed here can be supported by your circuit. This integration maps limited set of keys to circuit specific ebusd values.
   type: list
-  required: false
+  required: true
```

Comment:
This would require a code change; which isn't linked (but also isn't allowed anymore, as we don't allow changes to YAML configuration options at this point)

Reply:
It would not require a code change, it just documents existing behaviour. This is the whole reason for the change.